clang/lib/ExtractAPI/Serialization/SymbolGraphSerializer.cpp

```diff
@@ -38,14 +38,14 @@ namespace {
 
 /// Helper function to inject a JSON object \p Obj into another object \p Paren
 /// at position \p Key.
-void serializeObject(Object &Paren, StringRef Key, std::optional<Object> Obj) {
```

Comment:
Most of these changes involve renaming the `serialize*` functions defined within the anonymous namespace.